src/plugins/viewport/sim-host.js

```diff
@@ -0,0 +1,193 @@
+    { 'id': 'Bold9900', 'name': 'BlackBerry Bold 9900', 'width': 640, 'height': 480 },
+    { 'id': 'Curve9300', 'name': 'BlackBerry Curve 9300', 'width': 320, 'height': 240 },
+    { 'id': 'Curve9350-9360-9370', 'name': 'BlackBerry Curve 9350/9360/9370', 'width': 480, 'height': 360 },
+    { 'id': 'FWVGA', 'name': 'Generic - FWVGA (480x854)', 'width': 480, 'height': 854 },
+    { 'id': 'G1', 'name': 'HTC G1', 'width': 320, 'height': 480 },
+    { 'id': 'HPPre3', 'name': 'HP Pre 3', 'width': 480, 'height': 800 },
+    { 'id': 'HPVeer', 'name': 'HP Veer', 'width': 320, 'height': 400 },
+    { 'id': 'HVGA', 'name': 'Generic - HVGA (320x480)', 'width': 320, 'height': 480 },
+    { 'id': 'iPad', 'name': 'iPad', 'width': 768, 'height': 1024 },
+    { 'id': 'iPad3', 'name': 'iPad 3', 'width': 1536, 'height': 2048 },
+    { 'id': 'iPhone3', 'name': 'iPhone 3G/3Gs', 'width': 320, 'height': 480 },
+    { 'id': 'iPhone4', 'name': 'iPhone 4/4s', 'width': 640, 'height': 960 },
+    { 'id': 'iPhone5', 'name': 'iPhone 5', 'width': 640, 'height': 1136 },
+    { 'id': 'Legend', 'name': 'HTC Legend', 'width': 320, 'height': 480 },
+    { 'id': 'Nexus', 'name': 'Nexus One', 'width': 480, 'height': 800 },
+    { 'id': 'Nexus4', 'name': 'Nexus 4', 'width': 768, 'height': 1280 },
+    { 'id': 'Nexus7', 'name': 'Nexus 7 (Tablet)', 'width': 800, 'height': 1280 },
+    { 'id': 'NexusGalaxy', 'name': 'Nexus (Galaxy)', 'width': 720, 'height': 1280 },
+    { 'id': 'NexusS', 'name': 'Nexus S', 'width': 480, 'height': 800 },
+    { 'id': 'NokiaN8', 'name': 'Nokia N8', 'width': 360, 'height': 640 },
+    { 'id': 'NokiaN97', 'name': 'Nokia N97/5800 (touch)', 'width': 360, 'height': 640 },
+    { 'id': 'PalmPre', 'name': 'Palm Pre', 'width': 320, 'height': 480 },
+    { 'id': 'PalmPre2', 'name': 'Palm Pre 2', 'width': 320, 'height': 480 },
+    { 'id': 'Pearl9100', 'name': 'BlackBerry Pearl 9100', 'width': 360, 'height': 400 },
+    { 'id': 'Playbook', 'name': 'BlackBerry Playbook', 'width': 1024, 'height': 600 },
+    { 'id': 'Q10', 'name': 'BlackBerry Q10', 'width': 720, 'height': 720 },
+    { 'id': 'QVGA', 'name': 'Generic - QVGA (240X320)', 'width': 240, 'height': 320 },
+    { 'id': 'Style9670', 'name': 'BlackBerry Style 9670', 'width': 360, 'height': 400 },
+    { 'id': 'Tattoo', 'name': 'HTC Tattoo', 'width': 240, 'height': 320 },
+    { 'id': 'Torch9800', 'name': 'BlackBerry Torch 9800', 'width': 360, 'height': 480 },
+    { 'id': 'Torch9810', 'name': 'BlackBerry Torch 9810', 'width': 480, 'height': 640 },
+    { 'id': 'Torch9860-9850', 'name': 'BlackBerry Torch 9860/9850', 'width': 480, 'height': 800 },
+    { 'id': 'Wave', 'name': 'Samsung Wave', 'width': 480, 'height': 800 },
+    { 'id': 'WQVGA', 'name': 'Generic - WQVGA (240x480)', 'width': 240, 'height': 400 },
+    { 'id': 'WVGA', 'name': 'Generic - WVGA (480x800)', 'width': 480, 'height': 800 },
+    { 'id': 'Z10', 'name': 'BlackBerry Z10', 'width': 768, 'height': 1280 }
```

Comment:
We could handle this list of devices in a different way. This is metadata, there's no need to be defined here in the code, it can be a separate json or multiple's json files with all the required information. When more information is required, this will grow, even when we add new devices to support. Having them in separate files, or at leas in a single dedicated file will help for maintainability. 
For today this is OK, but in the future we will need a better handling of devices metadata.